frontend/src/app/containers/UserDetailPage/index.tsx

```diff
@@ -40,6 +42,69 @@ export function UserDetailPage(props: Props) {
 
   if (user === null) return <></>;
 
+  const addBanButton = () => {
```

Comment:
try to write this in an separate file, you can delete the old ban button if needed